extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -69,14 +71,24 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   unlayoutCallback() {
+    this.viewport_.updatePaddingBottom(0);
     return true;
   }
 
   /** @override */
   detachedCallback() {
+    this.viewport_.updatePaddingBottom(0);
     this.removeOnScrollListener_();
   }
 
+  /** @override */
+  collapsedCallback() {
+    this.collapse();
+    this.vsync_.mutate(() => {
+      this.viewport_.updatePaddingBottom(0);
```

Comment:
Not sure about this, `#collapse` function has a schedule `FOUR_FRAME_DELAY_` feature, and I am also doing the `vsync_` here. I feel I need a better way here